opensimplex/opensimplex.py

```diff
@@ -74,28 +74,28 @@ def _init(seed=DEFAULT_SEED):
 
 
 @njit(cache=True)
-def _extrapolate2(perm, xsb, ysb, dx, dy):
+def _extrapolate2(perm: np.ndarray, xsb: int, ysb: int, dx: float, dy: float) -> np.ndarray:
```

Comment:
Why are you mixing ints and floats here? Most everywhere else it's been floats

Reply:
Nvm, I compared it with the original C source functions and the inputs matched correctly :+1: